Formula/braid.rb

```diff
@@ -0,0 +1,21 @@
+class Braid < Formula
+  desc "Simple tool to help track vendor branches in a Git repository"
+  homepage "https://cristibalan.github.io/braid/"
+  url "https://github.com/cristibalan/braid.git",
+      :tag => "v1.0.21",
+      :revision => "8d953288babe6bf320a390113cbca0f314607272"
+  head "https://github.com/cristibalan/braid.git"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "braid.gemspec"
+    system "gem", "install", "braid-#{version}.gem"
+    bin.install libexec/"bin/braid"
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+  end
+
+  test do
+    output = shell_output("#{bin}/braid --help", 1)
```

Comment:
This should be a test that exercises, and verifies, the software's functionality once installed.